Validation/src/Validation/__main__.py

```diff
@@ -0,0 +1,97 @@
+"""CLI for comparison plots within LDMX Validation"""
+
+# standard
+import argparse
+import os
+import re
+import logging
+
+# external
+import matplotlib
+# this line allows us to run without an X server connected
+#    basically telling MPL that it will not open a window
+matplotlib.use('Agg')
+
+import matplotlib.pyplot as plt
+import mplhep
+plt.style.use(mplhep.style.ROOT)
+
+# us
+from ._differ import Differ
+from ._file import File
+from ._plotter import plotter
+
+# guard incase someone imports this somehow
+if __name__ == '__main__' :
+    parser = argparse.ArgumentParser("python3 -m Validation",
+        description="""
+        Make comparison plots between different files within a directory.
+
+        The labels of different plots within the directory is controlled by
+        the parameter you choose. The parameters of a file are extracted from
+        the file name by splitting the filename into key-val pairs separated 
+        by underscores (i.e. key1_val1_key2_val2_..._keyN_valN.root). If no
+        parameter is provided, then the first key/val is used.
+        """
+        )
+
+    parser.add_argument('data',help='directory of event and histogram files')
+    parser.add_argument('--log',help='logging level',choices=['info','debug','warn','error'], default='warn')
+    parser.add_argument('--label',help='label for grouping of data, defaults to data directory name')
+    parser.add_argument('--out-dir',help='directory to which to print plots. defaults to input data directory')
+    parser.add_argument('--systems',required=True, choices=plotter.__registry__.keys(), nargs='+',
+        help='list of plotters to run')
+    parser.add_argument('--param',help='parameter in filename to use as file labels')
```

Comment:
I think it would be quite useful if you weren't limited to one of the parameters. Sure, for comparing v12 and v14 usually we just want those two but I've found myself wanting to compare more than two files more than once.  From what I can tell, this should be relatively straight forward to do. e.g. split  `param` by some separator (e.g. `,`) or use a parameter type directly that supports multiple arguments.  Then change the check in in _file to check each entry rather than just one and join them into one label

Reply:
This makes sense - do you want to draft what you're thinking or should I try for it?

Reply:
Pushed an implementation now. Tested with both 0, 1, and 3 parameters and all worked 
[hcal_dqm_back_hit_multiplicity.pdf](https://github.com/LDMX-Software/ldmx-sw/files/11334390/hcal_dqm_back_hit_multiplicity.pdf)

Reply:
Now that I see that Github doesn't automatically show PDFs, maybe it would be worth making the output format configurable and default to pdf?